mlflow/ml-package-versions.yml

```diff
@@ -126,7 +126,7 @@ xgboost:
       ">= 0.0.0": ["scikit-learn"]
       "< 1.6": ["pandas<2"]
     run: |
-      pytest tests/xgboost/test_xgboost_model_export.py
+      dev/pytest.sh tests/xgboost/test_xgboost_model_export.py
```

Comment:
We have more pytest invocations. I'll wait for a day to ensure this change breaks nothing.